src/meds_torch/data/datamodule.py

```diff
@@ -155,6 +155,22 @@ def test_dataloader(self) -> DataLoader[Any]:
             **self.cfg.dataloader,
         )
 
+    def predict_dataloader(self) -> DataLoader[Any]:
+        """Create and return the predict dataloader.
+
+        :return: The predict dataloader.
+        """
+        if self.cfg.predict_dataset == "train":
```

Comment:
It seems like we should use one vernacular: Train, Tuning, and held out, or train, val, and test (and I would vote for the former as it is what MEDS uses. But, if lightning standardizes on `train`, `val`, and `test` then this is unavoidable.